module/designer/src/Domain/Entity/TemplateElement.php

```diff
@@ -1,5 +1,4 @@
 <?php
-
```

Comment:
👎🏻

Reply:
We should have 1 space between PHP tag and licence comment?